src/main/java/net/neoforged/neoforge/client/event/ClientTickEvent.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.client.event;
+
+import net.neoforged.bus.api.Event;
+
+/**
+ * Base class of the two client tick events.
```

Comment:
I think that it would be wise to link to `RenderFrameEvent` and explain the difference. Often you might be looking for the other one.

The most important difference that should be pointed out to beginners in the javadoc is that `ClientTickEvent` will fire for every "logical" tick, hopefully 20 times per second. The other one fires once per frame.